modules/lang-mustache/src/test/resources/rest-api-spec/test/lang_mustache/50_msearch.yaml

```diff
@@ -51,3 +51,4 @@
           - query:
               { "template": { "query": { "term": { "foo": { "value": "{{template}}" } } }, "params": { "template": "bar" } } }
   - match: { responses.0.hits.total: 1 }
+
```

Comment:
This particular test seems to be more testing msearch than mustache. I think it should stay in the main folder for rest tests minus the mustache part, and then we can have a dedicated mustache test for the msearch integration if we want?